src/browser.ts

```diff
@@ -115,13 +135,24 @@ export class GrpcClient {
         metadata,
         callback
       ) => {
-        languageServiceStub[methodName].apply(languageServiceStub, [
-          req,
-          callback,
-        ]);
+        const cancelHandler = languageServiceStub[methodName].apply(
+          languageServiceStub,
+          [req, callback]
+        ) as CancelHandler;
+        return {
+          cancel: () => {
+            if (cancelHandler.canceller === undefined) {
```

Comment:
Just `if (!cancelHandler.canceller)` makes it easier to read :)

Reply:
Done.